crates/sui-graphql-rpc/src/server/sui_sdk_data_provider.rs

```diff
@@ -175,6 +180,48 @@ impl DataProvider for SuiClient {
         Ok(connection)
     }
 
+    async fn fetch_checkpoint_connection(
+        &self,
+        first: Option<u64>,
+        after: Option<String>,
+        last: Option<u64>,
+        before: Option<String>,
+    ) -> Result<Connection<String, Checkpoint>> {
+        ensure_forward_pagination(&first, &after, &last, &before)?;
+
+        let count = first.map(|q| q as usize);
+        let pg = self
+            .read_api()
+            .get_checkpoints(
+                after.map(|x| {
+                    if let Ok(after_u64) = x.parse::<u64>() {
+                        SerdeBigInt::from(after_u64)
+                    } else {
+                        panic!("Cannot parse checkpoint id for after parameter into u64")
```

Comment:
![image](https://github.com/MystenLabs/sui/assets/332275/fc29d2ea-81b9-4e2b-90ac-ab92fd26ff02)

You'll bring down the whole service for an input error!

Reply:
Fixed